src/index.umd.ts

```diff
@@ -10,13 +10,15 @@ import * as Controls from "./control";
 import * as Loaders from "./loader";
 import * as Plugins from "./plugin";
 import * as Constants from "./const/external";
+import * as Static from "./static";
```

Comment:
How about this code?:
```js
import View3D, * as modules from "./index";

merge(View3D, modules);
export default View3D;
```

Reply:
Looks good, I'll change to that style if it works.